arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1146 @@
+        },
+        "LastUpdated": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the time of last update."
+        },
+        "ETag": {
+          "type": "string",
+          "description": "Gets or sets the ETag of the search results."
+        },
+        "sort": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSort"
+          },
+          "description": "Gets or sets sort."
+        },
+        "requestTime": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the request time."
+        },
+        "aggregatedValueField": {
+          "type": "string",
+          "description": "Gets or sets the aggregated value field."
+        },
+        "aggregatedGroupingFields": {
+          "type": "string",
+          "description": "Gets or sets the aggregated grouping fields."
+        },
+        "sum": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the sum."
+        },
+        "max": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the max."
+        },
+        "schema": {
+          "$ref": "#/definitions/MetadataSchema",
+          "description": "Gets or sets the schema."
+        }
+      },
+      "description": "Metadata for search results."
+    },
+    "SavedSearchValue": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id of the saved search."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets or sets the etag of the saved search."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SavedSearchProperties",
+          "description": "Gets or sets properties of the saved search."
+        }
+      },
+      "description": "Value object for saved search results."
+    },
+    "SearchListSavedSearchResult": {
+      "properties": {
+        "__metadata": {
+          "$ref": "#/definitions/SearchMetadata",
+          "description": "Gets or sets the metadata from search results."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SavedSearchValue"
+          },
+          "description": "Gets or sets the array of result values."
+        }
+      },
+      "description": "The saved search operation response."
+    },
+    "SearchError": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Gets or sets the error type."
+        },
+        "message": {
+          "type": "string",
+          "description": "Gets or sets the error message."
+        }
+      },
+      "description": "Details for a search error."
+    },
+    "SearchGetSearchResultsResponse": {
```

Comment:
Is this required? I noticed that  most resource providers are not using camel casing.